package.py

```diff
@@ -511,7 +511,8 @@ def prompt(ids, on_done):
             else:
                 client = JVMClient(
                     source_root(), host, int(port), backchannel_opts={
-                        "port": settings().get("clojure").get("backchannel").get("port")
+                        "port": settings().get("clojure").get("backchannel").get("port"),
+                        "bind_address": settings().get("clojure").get("backchannel").get("bind_address")
```

Comment:
It's awkward, but we have to hard-code the `localhost` default value here, too. That is:

```python
settings().get("clojure").get("backchannel").get("bind_address", "localhost")
```

Otherwise, if you have a block like this in your settings (like I did):

```json
  "clojure": {
    "backchannel": {
      "port": 0
    }
  }
```

`bind_address` will be `None`.

If you can push this change, I think everything will be in order and I can merge this PR.

Reply:
Ok, you are right